### ISSUE TITLE
Sensors will not load on version v2.4.5 - KeyError: 'balanced_consumption'

### ISSUE THREAD

**Issue by convicte** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#96]
```
This error originated from a custom integration.

Logger: custom_components.tauron_amiplus.coordinator
Source: custom_components/tauron_amiplus/statistics.py:54
Integration: Tauron AMIplus (documentation, issues)
First occurred: January 24, 2023 at 18:27:36 (8 occurrences)
Last logged: 09:35:46

Unexpected error fetching tauron_amiplus data: 'balanced_consumption'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 225, in _async_refresh
    self.data = await self._async_update_data()
  File "/config/custom_components/tauron_amiplus/coordinator.py", line 31, in _async_update_data
    await self.generate_statistics(data)
  File "/config/custom_components/tauron_amiplus/coordinator.py", line 37, in generate_statistics
    await statistics_updater.update_all(data)
  File "/config/custom_components/tauron_amiplus/statistics.py", line 54, in update_all
    stat = await self.get_stats(raw_data[v["data_source"]], s)
KeyError: 'balanced_consumption'
```

![image](https://user-images.githubusercontent.com/78666770/214518110-39144eaa-a546-441d-8661-cf7096f16e47.png)

**Comment by PiotrMachowski** [comment]
This error has been fixed (#94), it seems that you don't have `v2.4.5` installed, as the line number in log matches `v2.4.4`.

`v2.4.5`:
![image](https://user-images.githubusercontent.com/6118709/214558325-db21da58-72bf-4534-97fa-13e8139575cf.png)

`v2.4.4`:
![image](https://user-images.githubusercontent.com/6118709/214558416-a2c155ea-727e-4bdf-bdad-7dcf6998fb7f.png)

**Comment by convicte** [comment]
Thanks - worked like a charm!

I've checked in HACS, and it was showing me the latest version of 2.4.5, while it obviously wasn't on it.
After I 're-downloaded' the version, things came back up!
Sorry for the trouble.
